src/main/webapp/app/exercises/text/manage/text-exercise/text-exercise-detail.component.html

```diff
@@ -17,7 +17,7 @@ <h2><span jhiTranslate="artemisApp.textExercise.detail.title">Text Exercise</spa
                     <span jhiTranslate="artemisApp.textExercise.checkPlagiarism">Check Plagiarism</span>
                 </a>
             </div>
-            <div *ngIf="textExercise.releaseDate != undefined && moment(textExercise.releaseDate).isBefore(moment())" class="mt-3">
+            <div *ngIf="textExercise.releaseDate != undefined && dayjs(textExercise.releaseDate).isBefore(dayjs())" class="mt-3">
```

Comment:
I suggest not exposing the date API to the template

Reply:
I created a confluence page for general implementation/style aspects which can be handled separately after discussion how to proceed with your suggestions.